verde/utils.py

```diff
@@ -201,3 +203,49 @@ def maxabs(*args):
     arrays = [np.atleast_1d(i) for i in args]
     absolute = [np.abs([i.min(), i.max()]).max() for i in arrays]
     return np.max(absolute)
+
+
+def grid_to_table(grid):
+    """
+    Converts an xarray Dataset grid to a Pandas DataFrame
+
+    Parameters
+    ----------
+    grid
+        An xarray grid Dataset composed of X and Y coordinates and a variable.
```

Comment:
```suggestion
        A 2D grid with a single data variable.
```

The type is already specified in the line above.